src/arch/riscv/isa/decoder.isa

```diff
@@ -3070,9 +3070,11 @@ decode QUADRANT default Unknown::unknown() {
                         xc->getRegOperand(this, 2,
                             &tmp_s2);
                         auto Vs2bit = tmp_s2.as<vu>();
+                        uint32_t vlen = RiscvISA::ISA::getVecLenInBits();
```

Comment:
This should use the ISA pointer via `xc` (I think). I.e., it's theoretically possible to have cores with different length vectors.